content/installation/ElasticSearch/_index.md

```diff
@@ -47,54 +47,15 @@ toc = true
    Configuration syntax is explained [here](/configuration).
    Full Helm chart parameters list is documented [here](/configuration/helm-chart-parameters).
 
-  {{% notice note %}}
   With the default configuration, BotKube will watch all the resources in all the namespaces for _create_, _delete_ and _error_ events.<br>
   If you wish to monitor only specific resources, follow the steps given below:
-  {{% /notice%}}
-
-  - Create new file config.yaml and add resource configuration as described on the [configuration](/configuration) page.
-
-    (You can refer sample config from https://raw.githubusercontent.com/kubeshop/botkube/v0.12.4/helm/botkube/sample-res-config.yaml)
-
-  ```yaml
-  config:
-    ## Resources you want to watch
-    resources:
-    - name: v1/pods        # Name of the resource. Resource name must be in
-                           # group/version/resource (G/V/R) format
-                           # resource name should be plural
-                           # (e.g apps/v1/deployments, v1/pods)
-      namespaces:          # List of namespaces, "all" will watch all the namespaces
-        include:
-        - all
-        ignore:            # List of namespaces to be ignored, used only with include: all
-        - kube-system      # example : include [all], ignore [x,y,z]
-      events:              # List of lifecycle events you want to receive,
-                           # e.g create, update, delete, error OR all
-      - create
-      - delete
-      - error
-    - name: batch/v1/jobs
-      namespaces:
-        include:
-        - ns1
-        - ns2
-      events:
-      - create
-      - update
-      - delete
-      - error
-      updateSetting:
-        includeDiff: true
-        fields:
-        - spec.template.spec.containers[*].image
-        - status.conditions[*].type
-  ```
-  - Pass the YAML file as a flag to `helm install` command, e.g.:
 
-    ```bash
-    $ helm install --version v0.12.4 --name botkube --namespace botkube --create-namespace -f /path/to/config.yaml --set=...other args..
-    ```
+  1. Create new file config.yaml and add resource configuration as described on the [source](/configuration/source) page.
```

Comment:
Please add backticks:

```suggestion
  1. Create new file `config.yaml` and add resource configuration as described on the [source](/configuration/source) page.
```

I know Hugo theme renders them as items to copy, but with Docusaurus we won't have this problem 🙂